src/core/gc/config.d

```diff
@@ -39,6 +40,7 @@ struct Config
 
         printf("GC options are specified as white space separated assignments:
     disable:0|1    - start disabled (%d)
+    fork:0|1    - set fork behaviour (disabled by default) (%d)
```

Comment:
The user will probably not care how it's implemented (i.e. that is using fork), but selects the GC to be `concurrent`. So maybe the option should be the latter. Alternatively, it could also just be another GC named `concurrent`, but how to combine that with `precise`?